textpipe/pipeline.py

```diff
@@ -23,9 +24,16 @@ def __init__(self, pipeline=None, **kwargs):
         language: 2-letter code for the language of the text
         hint_language: language you expect your text to be
         """
-        self.pipeline = pipeline or kwargs.get('pipeline', None)
-        self.language = kwargs.get('language', None)
-        self.hint_language = kwargs.get('hint_language', None)
+        self.operations = operations
+        self.language = language
+        self.hint_language = hint_language
```

Comment:
I agree this is better than the `kwargs.get` approach, but if we move the language detection params, I propose not to make this part of this PR.

Reply:
If this is removed from the PR, then the json dump of a pipeline object will look something like this:
```
{
    "operations": [
        "NSentences",
        "CleanText"
    ],
    "language": "it",
    "hint_language": null,
    "kwargs": {
        "language": "it"
    }
}
```
I don't particularly like the duplication.